homeassistant/loader.py

```diff
@@ -78,6 +78,7 @@
     "repairs",
     "system_health",
     "trigger",
+    "backup",
```

Comment:
This list was sorted alphabetically.

Reply:
https://github.com/home-assistant/core/pull/133905